src/containers/audio-rewards-page/AudioRewardsPage.tsx

```diff
@@ -26,7 +27,7 @@ export const messages = {
 
 export const RewardsContent = () => {
   const wm = useWithMobileStyle(styles.mobile)
-
+  useRequiresAccount(TRENDING_PAGE)
```

Comment:
Pretty sure this page doesn't require an account? It's live right now without this...

Reply:
If will not error, but this page does show account specific information like your balance which may be confusing to see if you're not logged in. Thoughts @raymondjacobson - maybe we just trigger redirect on click send/receive/manage wallets similar to when you click favorite track when not logged in.

Reply:
since we're already live with the rewards page for unauthed users, this seems like a product question to me. let's rope in the team